apiserver/internal/handlers/pubsub/package_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package pubsub_test
+
+import (
+	"crypto/x509"
+	"net/http"
+	"testing"
+
+	"github.com/gorilla/websocket"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	jujuhttp "github.com/juju/juju/internal/http"
+	coretesting "github.com/juju/juju/internal/testing"
+)
+
+func TestPackage(t *testing.T) {
+	coretesting.MgoTestPackage(t)
```

Comment:
Was ist das? Do we still need mongo for these tests?

Reply:
Yeah, unfortunately. The test suite needs the APIServer test suite. I will sweep back around and clean this up, but it's a side quest I don't want to endure yet.